spec/foundation_rails_helper/flash_helper_spec.rb

```diff
@@ -83,4 +85,15 @@
       #   undefined method `css' for nil:NilClass
     end
   end
+
+  context "with (closable: false) option" do
+    it "doesn't display the close button" do
+      allow(self).to receive(:flash).and_return({ :success => "Flash message" })
+      node = Capybara.string display_flash_messages(closable: false)
+      expect(node).
+        to  have_css("div.flash.callout.success", :text => "Flash message").
+        and not_have_css("[data-close]", :text => "×")
```

Comment:
Ooh, now that I see this, the normal formatting of this is with the dots at the start:

```ruby
expect(node)
  .to have_css("div.flash.callout.success", :text => "Flash message")
  .and not_have_css("[data-close]", :text => "×")
```